packages/entity-browser/src/routes/detail/components/DetailForm/DetailForm.js

```diff
@@ -40,6 +42,7 @@ export class DetailForm extends React.Component {
     return (
       <Field
         key={key}
+        readOnlyForm={this.isReadOnlyForm}
```

Comment:
yes, it can. but we only want to show "formattedValues" if the whole form is readonly. Otherwise we want "editableValues" and if a single field is readonly it is disabled.